packages/editor-ui/src/components/layouts/ResourcesListLayout.vue

```diff
@@ -356,30 +315,13 @@ export default defineComponent({
 				}
 			});
 		},
-		resourcesNotOwned(): IResource[] {
-			return (this.resources as IResource[]).filter((resource) => {
-				return resource.ownedBy && resource.ownedBy.id !== this.usersStore.currentUser?.id;
-			});
-		},
-		shouldSwitchToAllSubview(): boolean {
-			return !this.hasFilters && this.isOwnerSubview && this.resourcesNotOwned.length > 0;
-		},
 	},
 	watch: {
-		isOwnerSubview() {
-			this.sendSubviewTelemetry();
-		},
 		filters(value) {
 			this.filtersModel = value;
 		},
-		'filtersModel.ownedBy'(value) {
-			if (value) {
-				this.setOwnerSubview(false);
-			}
-			this.sendFiltersTelemetry('ownedBy');
-		},
-		'filtersModel.sharedWith'() {
-			this.sendFiltersTelemetry('sharedWith');
+		'filtersModel.homeProject'() {
+			this.sendFiltersTelemetry('homeProject');
```

Comment:
Hey Csaba, please add the changes events to https://www.notion.so/n8n/List-of-telemetry-events-that-will-need-to-change-0359839a92444315b7c4030e07481ac4

Reply:
Actually, the argument the function gets is not being sent; it was just to check if it wasn't a filter reset event.
When resetting, there is no telemetry call.

Other than that it just sends the whole filter state which is still happening but with some changes (`ownedBy` and `sharedWith` are removed from the object and `homeProject` added)

But I added it to the notion list just in case